pkgs/by-name/go/goat/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+,
```

Comment:
```suggestion
, buildGoModule
, fetchFromGitHub
```

Reply:
Arrg, that bites me every time. :laughing: Wished the formatter would catch that...